platforms/gke/base/use-cases/inference-ref-arch/terraform/comfyui/src/custom-nodes/gemini/config.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a preview version of gemini custom node
+import requests
+
+
+# Fetch GCP project ID and zone required to authenticate with Vertex AI APIs
+def get_gcp_metadata(path):
```

Comment:
Have you considered moving this out into a shared module?

Reply:
Each custom node will branch out to a separate repo so it will need it's own config.py. I will start moving the custom nodes soon